snuba/datasets/configuration/transactions/storages/transactions.yaml

```diff
@@ -0,0 +1,216 @@
+      { name: user_name, type: String, args: { schema_modifiers: [nullable] } },
+      { name: user_email, type: String, args: { schema_modifiers: [nullable] } },
+      { name: sdk_name, type: String },
+      { name: sdk_version, type: String },
+      { name: http_method, type: String, args: { schema_modifiers: [nullable] } },
+      { name: http_referer, type: String, args: { schema_modifiers: [nullable] } },
+      {
+        name: tags,
+        type: Nested,
+        args:
+          {
+            subcolumns:
+              [
+                { name: key, type: String },
+                { name: value, type: String },
+              ],
+          },
+      },
+      { name: _tags_hash_map, type: Array, args: { type: UInt, arg: 64, schema_modifiers: [readonly] } },
+      {
+        name: contexts,
+        type: Nested,
+        args:
+          {
+            subcolumns:
+              [
+                { name: key, type: String },
+                { name: value, type: String },
+              ],
+          },
+      },
+      {
+        name: measurements,
+        type: Nested,
+        args:
+          {
+            subcolumns:
+              [
+                { name: key, type: String },
+                { name: value, type: Float, args: { size: 64 } },
+              ],
+          },
+      },
+      {
+        name: span_op_breakdowns,
+        type: Nested,
+        args:
+          {
+            subcolumns:
+              [
+                { name: key, type: String },
+                { name: value, type: Float, args: { size: 64 } },
+              ],
+          },
+      },
+      {
+        name: spans,
+        type: Nested,
+        args:
+          {
+            subcolumns:
+              [
+                { name: op, type: String },
+                { name: group, type: UInt, args: { size: 64 } },
+                { name: exclusive_time, type: Float, args: { size: 64 } },
+                { name: exclusive_time_32, type: Float, args: { size: 32 } },
+              ],
+          },
+      },
+      { name: partition, type: UInt, args: { size: 16 } },
+      { name: offset, type: UInt, args: { size: 64 } },
+      { name: message_timestamp, type: DateTime },
+      { name: retention_days, type: UInt, args: { size: 16 } },
+      { name: deleted, type: UInt, args: { size: 8 } },
+      { name: type, type: String, args: { schema_modifiers: [readonly] } },
+      { name: message, type: String, args: { schema_modifiers: [readonly] } },
+      { name: title, type: String, args: { schema_modifiers: [readonly] } },
+      { name: transaction_source, type: String },
+      { name: timestamp, type: DateTime, args: { schema_modifiers: [readonly] } },
+      { name: group_ids, type: Array, args: { type: UInt, arg: 64 } },
+      { name: app_start_type, type: String },
+    ]
+  local_table_name: transactions_local
+  dist_table_name: transactions_dist
+  part_format: [retention_days, date]
+
+query_processors:
+  -
```

Comment:
This might be a stupid question but is there a specific reason why the processors are always defined from the next line of hyphen? It kind of looks unintuitive when reading the file

Reply:
Not a stupid question. I think we did that initially to try and line up the `processor` and `args` more but I'm also not a fan. I changed it.